api/v3/Mailing.php

```diff
@@ -43,6 +43,9 @@
  * @throws \Civi\API\Exception\UnauthorizedException
  */
 function civicrm_api3_mailing_create($params) {
+  if (isset($params['template_options']) && is_array($params['template_options'])) {
+    $params['template_options'] = $params['template_options'] === array() ? '{}' : json_encode($params['template_options']);
```

Comment:
This line would be more readable with parentheses around 
($params['template_options'] === array())

I found it harder to read than a normal ternery because of all the ( and { noise